specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/stable/2021-04-01/examples/createTenant.json

```diff
@@ -0,0 +1,67 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "api-version": "2021-04-01",
+    "resourceGroupName": "contosoResourceGroup",
+    "resourceName": "contoso.onmicrosoft.com",
+    "createTenantRequestBody": {
+      "location": "United States",
+      "sku": {
+        "name": "Standard",
+        "tier": "A0"
+      },
+      "properties": {
+        "createTenantProperties": {
+          "displayName": "Contoso",
+          "countryCode": "US"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "type": "Microsoft.AzureActiveDirectory/b2cDirectories",
+        "sku": {
+          "name": "Standard",
+          "tier": "A0"
+        },
+        "properties": {
+          "billingConfig": {
+            "billingType": "MAU"
+          },
+          "tenantId": "00000000-0000-0000-0000-000000000000"
+        },
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/contosoResourceGroup/providers/Microsoft.AzureActiveDirectory/b2cDirectories/contoso.onmicrosoft.com",
+        "name": "contoso.onmicrosoft.com",
+        "location": "United States",
+        "tags": null
+      }
+    },
+    "201": {
+      "body": {
+        "type": "Microsoft.AzureActiveDirectory/b2cDirectories",
+        "sku": {
+          "name": "Standard",
+          "tier": "A0"
+        },
+        "properties": {
+          "billingConfig": {
+            "billingType": "MAU"
+          },
+          "tenantId": "00000000-0000-0000-0000-000000000000"
+        },
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/contosoResourceGroup/providers/Microsoft.AzureActiveDirectory/b2cDirectories/contoso.onmicrosoft.com",
+        "name": "contoso.onmicrosoft.com",
+        "location": "United States",
+        "tags": null
+      }
+    },
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/providers/Microsoft.AzureActiveDirectory/operations/99999999-9999-9999-9999-999999999999",
```

Comment:
>Microsoft.AzureActiveDirectory/operations/99999999-9999-9999-9999-999999999999 [](http://example.com/codeflow?start=111&length=78)

In externalIdentities.json this path is defined to return Azure-AsyncOperation resource, which is for Azure-AsyncOperation header not Location header.

Location header URI must return the b2cDirectories resource if async PUT succeeded.

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#azure-asyncoperation-resource-format](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#azure-asyncoperation-resource-format)